code/_compile_options.dm

```diff
@@ -43,7 +43,9 @@
 // #define TRACK_MAX_SHARE //Allows max share tracking, for use in the atmos debugging ui
 #endif //ifdef TESTING
 
-//#define UNIT_TESTS //If this is uncommented, we do a single run though of the game setup and tear down process with unit tests in between
+// If this is uncommented, will attempt to load and initialize prof.dll/libprof.so.
+// We do not ship byond-tracy. Build it yourself here: https://github.com/mafemergency/byond-tracy/
+#define USE_BYOND_TRACY
```

Comment:
Should be commented by default due to this not being shipped out of the box (and even if this were included in the codebase, there's the performance overhead as well; the warn for Tracy being enabled exists for good reason!)

Reply:
oh, I was going to have it disabled in the final PR. I just had it uncommented because I was doing tests with it, and someone else wanted to see how I got it working.